CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+
+# Contributing
+
+We welcome and appreciate contributions from the community. Whether it's a bug report, feature request,
+code contribution, or anything else, we're grateful for your effort. Here's a brief guide to get you started.
+
+Any contribution you make will be reflected on [CONTRIBUTORS](CONTRIBUTORS.md).
+
+To get an overview of the project, read the [README](README.md).
+
+## Product Vision
+
+[Project Name] is a [description of the project]. Our goal is to [outline the project's goal].
```

Comment:
@SlnPons I would need you for this section ;)

Reply:
@Thykof done, just pushed my suggestion :)